### PR TITLE
Add wcag suffix for page title

### PR DESCRIPTION
**Pull request by ateamcms**
## Changes description & testing suggestions
To enable wcag suffix add to platform config (to flags list) new property wcagSuffix with text of this suffix

## Developers checklist
- [ ] *Check that dusk tests are working locally on compatible branch*
- [ ] *Mobile version of changes is developed* - if its a webshop feature, mobile version for this feature is developed

## QA checklist
- *Check regress in implementations custom css* - changes are not breaking other implementations designes
- Feature is tested in different screen sizes - desktop, mobile
- WCAG requirements are met - new feature is accessible by keyboard, there are an alt texts
- Translations are done

### CODE REVIEW

**Comment by lexlog** [comment]
- [ ] @ateamcms could you please also add example of suffix configuration to qdt-env example

**Comment by ateamcms** [comment]
added page title suffix to cms, also there is pr on backend https://github.com/teamforus/forus-backend/pull/2256

File: react/src/dashboard/components/pages/implementations-cms/ImplementationsCms.tsx [hunk 1]
```diff
@@ -393,6 +396,54 @@ export default function ImplementationsCms() {
                                     </div>
                                     <FormError error={form.errors.description} />
                                 </div>
+                                <div className="form-group form-group-inline form-group-inline-xl tooltipped">
```
Comment (dev-rminds) [inline react/src/dashboard/components/pages/implementations-cms/ImplementationsCms.tsx:399]: @ateamcms could you please replace this info block with this component? 
[react/src/dashboard/components/elements/forms/elements/FormGroupInfo.tsx](https://github.com/teamforus/forus-frontend-react/blob/develop/react/src/dashboard/components/elements/forms/elements/FormGroupInfo.tsx)

Comment (ateamcms) [reply react/src/dashboard/components/pages/implementations-cms/ImplementationsCms.tsx:400]: replaced

**Review by irinaBerendeeva87 — DISMISSED** [review]
### Checked:
✅ Browser tab postfix
Screenshot 2024-07-18 at 11 44 59

**Titles in Nijmegen**
✅ /fondsen-aanvraag: 
✅ Login page 

**Titles in Noordoostpolder**
✅ Login page 
✅ Aanvragen( /fondsen-aanvraag):
✅ Meedoenpakket: 
✅/aanbod/203: Aanbod 

**Titles in Goeree-Overflakkee**
✅ Login page
✅ Aanbod 
✅ Aanvragen( /fondsen-aanvraag)

**Review by lexlog — CHANGES_REQUESTED** [review]
https://github.com/teamforus/forus-frontend-react/pull/163#issuecomment-2236166544

**Review by lexlog — APPROVED** [review]

**Pull request merged**